scripts/presubmit.sh

```diff
@@ -97,27 +97,31 @@ main() {
     go build ${goflags} ./...
 
     echo 'running go test'
+    # Install test deps so that individual test runs below can reuse them.
+    echo 'getting test deps'
+    go get -t
+    echo 'installed test deps'
+    go test -i ./...
 
+    if [[ ${coverage} -eq 1 ]]; then
```

Comment:
Indentation is a bit funky (though it does make the diff a bit clearer).

Reply:
Fixed, I think